build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.1.0-alpha03'
+        classpath 'com.android.tools.build:gradle:4.1.0-alpha08'
```

Comment:
Have you tried making a build off your commit in jitpack? I found that on `alpha08` META-INF is missing after build and you cannot use this library. Related to [this](https://stackoverflow.com/questions/61450260/extension-function-cant-be-imported-from-library)

Reply:
You're right, I think Gradle Module Metadata file would be needed - https://blog.gradle.org/gradle-metadata-1.0.
Reverted the version for now. But.. The method is not visible for import in an external project, but the symbol is there, compiled (with 4.1.0-alpha08 it's not even accessible/compiled). META-INF is missing also. Do you have any ideas?